internal/scm-clients/clients/clients.go

```diff
@@ -35,11 +35,18 @@ func FetchClientData(accessToken string, repoUrl string) (*checkmodels.AssetsDat
 	repo, _ := adapter.GetRepository(orgName, repoName)
```

Comment:
I think you need to pass the branch parameter to GetRepository because there are a few more fetching that takes the branch into consider like isRepositoryContainsSecurityMdFile